server/src/main/java/com/adobe/testing/s3mock/S3MockApplication.java

```diff
@@ -207,8 +201,8 @@ public int getHttpPort() {
    *
    * @param keyRef A KMS Key Reference
    */
-  public void registerKMSKeyRef(final String keyRef) {
-    kmsKeyStore.registerKMSKeyRef(keyRef);
+  public void registerKmsKeyRef(final String keyRef) {
```

Comment:
This is part of the public API that people are using. Let’s not change this. Maybe we should ignore that rule completely, as some of the other changes to comply with it seem a bit forced (e.g. for SSLContext, or the renaming of the *IT classes to *It.

Reply:
Right breaking the interface is of course a bad idea. I personally like the style not having to or more upper case letter, also for IntegrationTests. However, I revert the interface changes and the *IT stuff. Unfortunately we will end up not using Google Standard Checks but create and maintain our own